src/object.d

```diff
@@ -882,6 +882,21 @@ nothrow @safe @nogc unittest
     }
 }
 
+// Test to ensure proper behavior of `nothrow` destructors
```

Comment:
Please reference the buzilla issue number, otherwise looks good to go.

Reply:
Thanks.